src/Microsoft.AspNet.OData/GlobalSuppressions.cs

```diff
@@ -57,4 +57,6 @@
 [assembly: SuppressMessage("Microsoft.Design", "CA1004:GenericMethodsShouldProvideTypeParameter", Scope = "member", Target = "Microsoft.AspNet.OData.Builder.DerivedTypeConstraintConfiguration.#Add`1()")]
 [assembly: SuppressMessage("Microsoft.Design", "CA1004:GenericMethodsShouldProvideTypeParameter", Scope = "member", Target = "Microsoft.AspNet.OData.Builder.DerivedTypeConstraintConfiguration.#AddConstraint`1()")]
 [assembly: SuppressMessage("Microsoft.Design", "CA1004:GenericMethodsShouldProvideTypeParameter", Scope = "member", Target = "Microsoft.AspNet.OData.Builder.ActionConfiguration.#HasDerivedTypeConstraintForReturnType`1()")]
-[assembly: SuppressMessage("Microsoft.Design", "CA1004:GenericMethodsShouldProvideTypeParameter", Scope = "member", Target = "Microsoft.AspNet.OData.Builder.FunctionConfiguration.#HasDerivedTypeConstraintForReturnType`1()")]
\ No newline at end of file
+[assembly: SuppressMessage("Microsoft.Design", "CA1004:GenericMethodsShouldProvideTypeParameter", Scope = "member", Target = "Microsoft.AspNet.OData.Builder.FunctionConfiguration.#HasDerivedTypeConstraintForReturnType`1()")]
+[assembly: SuppressMessage("Microsoft.Usage", "CA1801:ReviewUnusedParameters", MessageId = "readContext", Scope = "member", Target = "Microsoft.AspNet.OData.Formatter.Deserialization.DeserializationHelpers.#ApplyInstanceAnnotations(System.Object,Microsoft.AspNet.OData.Formatter.Deserialization.ODataResourceWrapper,Microsoft.OData.Edm.IEdmStructuredTypeReference,Microsoft.AspNet.OData.Formatter.Deserialization.ODataDeserializerContext)")]
+[assembly: SuppressMessage("Microsoft.Design", "CA1024:UsePropertiesWhereAppropriate", Scope = "member", Target = "Microsoft.AspNet.OData.Builder.IODataInstanceAnnotationContainer.#GetAllResourceAnnotations()")]
```

Comment:
Is this suppression still required, or is it leftover from a previous iteration? It looks like the current implementation of ApplyInsanceAnnotations does correctly use all of the input parameters.